helm/botkube/README.md

```diff
@@ -45,6 +45,7 @@ serviceAccount:
 | communications.elasticsearch.password | string | `"ELASTICSEARCH_PASSWORD"` |  |
 | communications.elasticsearch.server | string | `"ELASTICSEARCH_ADDRESS"` |  |
 | communications.elasticsearch.username | string | `"ELASTICSEARCH_USERNAME"` |  |
+| communications.elasticsearch.skipTLSVerify | boolean | `false` | Skip tls verification of elastic search node |
```

Comment:
Let's make it `bool`  that's how it would be everywhere else in the Readme :)

Reply:
Done :D